lec06-exploratory-data-analysis.Rmd

```diff
@@ -286,36 +286,40 @@ method – Refers to method used in imputation. We used predictive mean matching
 
 Then, using these 5 imputed datasets, mice creates a dataset after imputation with the following code:
 ```{r}
-# imputed <- complete(imputed)
+imputed <- complete(imputed)
 ```
 
 Now let's double check for missing values:
 ```{r}
-# imputed %>% 
-#   select(hindfoot_length, weight) %>% 
-#   map(~sum(is.na(.))) #taken from tidyverse library purr; note requirement to include . in the function
+imputed %>%
+  select(hindfoot_length, weight) %>%
+  map(~sum(is.na(.))) #taken from tidyverse library purr; note requirement to include . in the function
 ```
 
-OK, we've successfully imputed values for ``hindfoot_length`` and ``weight`` using multiple imputation! How can we check the accuracy of our imputation? One way is to compare our imputed means to our actual means (i.e. with missing data included). If the variables are meaningfully correlated to one another, the averages of the imputed variables should be close to the averages of our original variables.
+OK, we've successfully imputed values for `hindfoot_length` and `weight` using multiple imputation! How can we check the accuracy of our imputation? One way is to compare our imputed means to our actual means (i.e. with missing data included). If the variables are meaningfully correlated to one another, the averages of the imputed variables should be close to the averages of our original variables.
 ```{r}
-# actual <- survey$hindfoot_length
-# predicted <- imputed$hindfoot_length
-# mean(actual, na.rm=TRUE)
-# mean(predicted)
-# 
-# actual <- survey$weight
-# predicted <- imputed$weight
-# mean(actual, na.rm=TRUE)
-# mean(predicted)
+actual <- survey$hindfoot_length
+predicted <- imputed$hindfoot_length
+mean(actual, na.rm=TRUE)
+mean(predicted)
+
+actual <- survey$weight
+predicted <- imputed$weight
+mean(actual, na.rm=TRUE)
+mean(predicted)
 ```
 
-##Challenge
+## Challenge
 Let's add some NA's to the iris (native to R) dataset (copy the code below). Then use the MICE package to run multiple imputation on our new dataset. 
 ```{r}
-# iris2 <- iris
-# set.seed(10)
-# iris2[sample(1:nrow(iris2), 20), "Sepal.Length"] <- NA
-# iris2[sample(1:nrow(iris2), 20), "Petal.Length"] <- NA
+iris2 <- iris
+set.seed(10)
+iris2[sample(1:nrow(iris2), 20), "Sepal.Length"] <- NA
+iris2[sample(1:nrow(iris2), 20), "Petal.Length"] <- NA
 ```
 
-
+```{r, show=FALSE}
+unloadNamespace("mice")
+unloadNamespace("mitml")
+unloadNamespace("jomo")
```

Comment:
just remove this line

Reply:
Can't; [breaks lec9](https://travis-ci.org/UofTCoders/rcourse/builds/433090310#L2870).